code/modules/games/cah.dm

```diff
@@ -0,0 +1,262 @@
+	"The ClockWorks on a Tuesday night",
+	"Fucking synths",
+	"Man-eating purple pod plants",
+	"Filling fire extinguishers with hydrogen",
+	"Librarians",
+	"Squids",
+	"Cats",
+	"Lizards",
+	"Apes",
+	"Trees",
+	"Ants",
+	"Supermatter undergarments",
+	"Bluespace",
+	"Vending machine tech singularity",
+	"Kitchen utensil anomalies",
+	"Locked lockers",
+	"Energy spatulas",
+	"Flashbang shoes",
+	"Space whales",
+	"Space sharks",
+	"Augmented male enhancers",
+	"Gyrating barks",
+	"Forehead-mounted laser weaponry",
+	"Pelvis-mounted laser weaponry",
+	"Cardboard cutouts",
+	"An obscene amount of rubber ducks",
+	"Brain cakes",
+	"A drone's game of life.",
+	"Anomaly suits",
+	"A hydrofarmer smoking reishi",
+	"Military-grade baseball bats",
+	"Barricading your classroom",
+	"An irritatingly chipper robot",
+	"Androids hanging out at a bar drinking beer",
+	"Flannel lingerie",
+	"A seventeen-year-old medical doctor",
+	"The throbbing erection that the police chief gets at the thought of shooting something",
+	"Trying to stab someone and hugging them instead",
+	"Waking up naked in a dark hallway",
+	"Horrible cloning accidents",
+	"Licking supermatter crystals due to a dare",
+	"Your uncle who WON'T STOP buying guns",
+	"Teaching a synthetic the Birds and the Bees",
+	"Unnecessary surgery",
+	"'Youth enhancements'",
+	"My addiction to spiders",
+	"DOCTOR MAXMAN!",
+	"A group of Skrell getting their squish on",
+	"Enough soporific to put the entire apartment block down",
+	"Delicious New Hai Phongese cuisine",
+	"Glorsh-Omega",
+	"Miranda Trasen",
+	"Michael 'The Mightinator' Frost",
+	"Aeson and Lopez-Zhang's love child",
+	"Liqin Hsiao-Li's dungeon",
+	"Mason's Trasen harem",
+	"Lydumila Zavodskoi having an identity crisis",
+	"Thanks, Dorn",
+	"His Imperial Majesty Emperor Boleslaw Keeser",
+	"Humping in a spacesuit",
+	"Planetary genocide",
+	"Sol III marijuana, baby",
+	"Eli's Extravagant Enchiladas",
+	"Racism",
+	"Interspecific orgies",
+	"Punching everyone in their genitals",
+	"Legalised bestiality",
+	"Balding at 20",
+	"A worthless degree",
+	"Overly insistent employees",
+	"Faulty mechanics",
+	"Swarms of flesh-eating nanites",
+	"Scientists",
+	"Sexy, Sexy Androids",
+	"Giant carnivorous plants",
+	"Spraying lube on it",
+	"Librarians",
+	"Unevolved humanoids",
+	"Small Trees",
+	"My special rocket ship underwear",
+	"Artificial intelligence",
+	"500 robotic spiders",
+	"500 sex bots",
+	"Cable ties",
+	"Getting sprayed by the soda fountain",
+	"RGB 'gamer-grade' drink coasters",
+	"Sentient cutlery",
+	"Locked garment protection box",
+	"Fancy spatulas",
+	"Slappy slap hand gloves",
+	"Sea bass",
+	"Fax machines",
+	"20th-century fountain pens",
+	"50% functional viagra",
+	"Questionably multipurpose lubricant",
+	"Eye lasers",
+	"Dick lasers",
+	"A bicycle",
+	"An actual goddamn time machine",
+	"That special OTP generator in your bookmarks",
+	"Affectionate spammers",
+	"Electron-positron annihilation",
+	"Harley O'Ryan",
```

Comment:
Immersion breaking to be joking about murders that happened less than three years ago.